src/main/java/org/dice_research/opal/launuts/lau/ExcelParser.java

```diff
@@ -0,0 +1,190 @@
+package org.dice_research.opal.launuts.lau;
```

Comment:
I guess this file should be inside org.dice_research.opal.launuts and should not be inside lau package because its a generic class both for nuts and lau.

Reply:
It's not generic to nuts for now, as basically its a replacement to lauCSV Parser, which was under Lau